indexd_test_utils/__init__.py

```diff
@@ -149,18 +149,12 @@ def indexd_server():
     hostname = 'localhost'
     port = 8001
     debug = False
-
-    indexd = Process(
-        target=app.run,
-        args=(hostname, port),
-        kwargs={'debug': debug},
-    )
-    indexd.start()
+    t = threading.Thread(target=app.run, kwargs={'host': hostname, 'port': port, 'debug': debug})
+    t.setDaemon(True)
+    t.start()
```

Comment:
I did a couple of tests mainly to check if the server stopped successfully after running any test, and it behaved correctly thanks to the `setDaemon`. But, it now means that this solution will only work for `session` scoped fixtures since there is no explicitly defined way to stop the thread. Luckily this is a `session` scoped fixture which I don't see changing any time soon, so this is good enough for me

Reply:
Should this information be included as a comment in the code?

Reply:
I think it's explicit in the code.